tempesta_fw/t/bomber.c

```diff
@@ -76,66 +76,68 @@ MODULE_LICENSE("GPL");
  * state and status. SsProto.type field is used here to store the index into
  * that array that can be passed around between callbacks.
  */
-typedef struct bmb_desc {
+struct bmb_conn {
 	SsProto		proto;
 	struct sock	*sk;
 	uint32_t	flags;
-} TfwConDesc;
+};
```

Comment:
Previous version was better - we use typedef to avoid using ```struct``` keyword in all definitions.